openshift/openshift-copies.go

```diff
@@ -690,9 +690,11 @@ func restClientFor(config *restConfig) (*url.URL, *http.Client, error) {
 		return nil, nil, err
 	}
 
-	var httpClient *http.Client
+	httpClient := &http.Client{
+		Timeout: 1 * time.Minute,
+	}
 	if transport != http.DefaultTransport {
-		httpClient = &http.Client{Transport: transport}
+		httpClient.Transport = transport
 	}
 
```

Comment:
While `openshift-copies.go` _is_ modified from the originals, I’d prefer not introducing new behavior in there. Could this be moved to `openshift.go` (which contains the only caller of this function)?